crates/node-db/tests/create.rs

```diff
@@ -6,9 +6,7 @@ mod util;
 #[test]
 fn create_tables() {
     let mut conn = test_conn();
-    let tx = conn.transaction().unwrap();
-    node_db::create_tables(&tx).unwrap();
-    tx.commit().unwrap();
+    let _ = node_db::with_tx(&mut conn, |tx| node_db::create_tables(tx));
```

Comment:
```suggestion
    node_db::with_tx(&mut conn, |tx| node_db::create_tables(tx)).unwrap();
```
I think we want to unwrap here to reflect the original behaviour of the test (i.e. test should fail if this fails).